modules/salt/pages/salt-states.adoc

```diff
@@ -164,3 +165,65 @@ Apply the highstate to all minions in the group:
 ----
 salt -I 'group_ids:GID' state.apply
 ----
+
+
+
+[[salt.pillars.setpackageendpoint]]
+== Use pillars to set the package download endpoint
+
+By default, {productname} assumes that the download endpoint to use is the FQDN of the {productname} server, or the {productname} Proxy.
+However, there are some cases where you might like to use a different FQDN as the download endpoint.
+The most common example is if you need to use load balancing, caching proxies, or in environments with complicated networking requirements.
+
+To change the package download endpoint, you can manually adjust three salt pillars:
+* [systemitem]``pkg_download_point_protocol``, defaults to [systemitem]``https``.
+* [systemitem]``pkg_download_point_host``, defaults to the FQDN of the {productname} Server (or Proxy, if in use).
+* [systemitem]``pkg_download_point_port``, defaults to [systemitem]``443``.
+
+If you do not adjust these pillars directly, {productname} will fall back to the default values.
+
+.Procedure: Changing the package download endpoint pillar
+
+. Navigate to [path]``/srv/pillar/`` and create a file called [filename]``top.sls``  with these contents:
++
+----
+base:
+  '*':
+    - pkg_download_endpoints
+----
++
+This example directs Salt to look at the [filename]``pkg_download_endpoints.sls`` file to determine the base URL to use.
+You can adjust this file to target different minions or groups, depending on your environment.
++
+. Remain in [path]``/srv/pillar/`` and create a file called [filename]``pkg_download_endpoints.sls`` with the base URLs you want to use.
+For example:
++
+----
+pkg_download_point_protocol: http
+pkg_download_point_host: example.com
+pkg_download_point_port: 444
+----
+. OPTIONAL: You can use grains to set conditional values, for example:
+----
+{% if grains['fqdn'] == 'minion1.example.com' %}
+    pkg_download_point: example1.com
+{% elif grains['fqdn'] == 'minion2.example.com'' %}
+    pkg_download_point: example2.com
+{%else%}
```

Comment:
```suggestion
{% else %}
```

Reply:
and also here this:

```
-{% elif grains['fqdn'] == 'minion2.example.com'' %}
+{% elif grains['fqdn'] == 'minion2.example.com' %}
```

Reply:
Wondering how do you create this nice inline patches?

Reply:
It's a GitHub feature in beta https://github.blog/changelog/2018-10-16-suggested-changes/